module_3/Dao/src/main/java/com/epam/ems/dto/fields/Constants.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.ems.dto.fields;
+
+public class Constants {
```

Comment:
Naming convention tells us NOT to create classes in plural form e.g. user instead of users. Constant instead of Constants